ephios/core/signup/participants.py

```diff
@@ -114,6 +114,9 @@ def reverse_event_detail(self, event):
 
     @property
     def icon(self):
+        title = _(
+            "Placeholder"
+        )  # not getting marked for translation when inside f-string for some reason
```

Comment:
yeah I think that's known and django suggests to use the old syntax

Reply:
seems to be a limitation in gettext